xGame/client/src/Components/MatchSelect/MatchCreateDialog.js

```diff
@@ -1,109 +1,174 @@
+                                            name="opponentValue"
+                                            label="Search for a user"
+                                            value={values?.["opponentValue"] || ""}
+                                            onChange={(e) => setFieldValue("opponentValue", e.target.value)}
+                                        />
+                                    </Col>
+                                    <Col xs="3" className="d-flex justify-content-center">
+                                        <Button
+                                            color="primary"
+                                            variant="contained"
+                                            style={{ textTransform: "none" }}
+                                            onClick={formProps.handleSubmit}
+                                        >
+                                            Search
+                                        </Button>
+                                    </Col>
+                                </Row>
+                                <br />
                             </Col>
-                          </Row>
-                          <br></br>
-                          <Row>
+                        );
+                    }}
+                </Formik>
+                <Formik
+                    onSubmit={async (values) => {
+                        const match = await matchService.createMatch(values);
+                        if (match?.error) setSearchError(match?.message);
+                        setShowMatchCreation(false);
+                    }}
+                    initialValues={{
+                        blackID: "",
+                        whiteID: activeUser?.id
+                    }}
+                >
+                    {(formProps) => {
+                        const { setFieldValue, values } = formProps;
+                        return (
                             <Col>
-                              <div role="group" aria-labelledby="my-radio-group">
-                                  <p>
-                                  <div>Your starting color: {values.picked}</div>
-                                  <label>
-                                    <Field type="radio" name="picked" value="Black" />
-                                    Black
-                                  </label>
-                                  <br></br>
-                                  <label>
-                                    <Field type="radio" name="picked" value="White" />
-                                    White
-                                  </label>
-                                  </p>
-                                </div>
+                                <Row className="pt-2">
+                                    <Col className="d-flex justify-content-center">
+                                        <Select

```

Comment:
This isn't a big deal since we're not really getting graded on our UI, but Material's Data Grid is a much better way to display search results than using a Select. I don't think you need to worry about changing it unless you want to though